spec/models/osbb_spec.rb

```diff
@@ -0,0 +1,68 @@
+require 'rails_helper'
+
+RSpec.describe Osbb, type: :model do
+  let!(:osbb) { FactoryBot.build(:osbb) }
+
+  it 'has a valid factory' do
+    expect(FactoryBot.build(:osbb)).to be_valid
```

Comment:
you can use :osbb

Reply:
Where use :osbb?

Reply:
@sviatoslav-krupa `expect (osbb).to be_valid `instead  `expect(FactoryBot.build(:osbb)).to be_valid`?

Reply:
I changed line 6)

Reply:
Exactly!